extensions/markdown-language-features/src/languageFeatures/copyFiles/dropIntoEditor.ts

```diff
@@ -84,14 +89,23 @@ export function createUriListSnippet(document: vscode.TextDocument, uris: readon
 
 		const ext = URI.Utils.extname(uri).toLowerCase().replace('.', '');
 		const insertAsImage = typeof options?.insertAsImage === 'undefined' ? imageFileExtensions.has(ext) : !!options.insertAsImage;
-
-		snippet.appendText(insertAsImage ? '![' : '[');
-
-		const placeholderText = options?.placeholderText ?? (insertAsImage ? 'Alt text' : 'label');
-		const placeholderIndex = typeof options?.placeholderStartIndex !== 'undefined' ? options?.placeholderStartIndex + i : undefined;
-		snippet.appendPlaceholder(placeholderText, placeholderIndex);
-
-		snippet.appendText(`](${mdPath})`);
+		const insertAsVideo = videoFileExtensions.has(ext);
+
+		if (insertAsVideo) {
+			snippet.appendText(`<video src="${mdPath}" placeholder="${mdPath}" autoplay loop controls muted title="`);
```

Comment:
For the `poster`, I know this is what some folks have been doing in the docs, but AFAIK setting a video as at the poster doesn't actually do the right thing. Let's leave it out

Also the other attributes are too aggressive for many use cases. Maybe just `controls `